### ISSUE TITLE
Question

### ISSUE THREAD

**Issue by mapageKA21** [miguelgimenezgimenez/cityViceServer#1]
This cityVice is like GTA? you can do anything? like a video game?

**Comment by miguelgimenezgimenez** [comment]
yes i used to work with Ian Salt, the famous game developer and he showd me
a lot of things.

2016-12-23 21:40 GMT+01:00 Manel <notifications@github.com>:

**Comment by mapageKA21** [comment]
Amazing
